ISISScientificComputing/autoreduce
# Add plot_location use to webapp

### Summary of work
* This should make the webapp run without commenting out the plotting code. 

### How to test your work
* Inspect the webapp to see if an image is displayed.
* Currently this should be possible for any MARI runs, although remember data is wiped from the dev node twice a day so if the image does not appear, you may need to re-run.


**Before merging ensure the release notes have been updated**
